azurerm/resource_arm_key_vault.go

```diff
@@ -232,19 +175,9 @@ func resourceArmKeyVaultCreate(d *schema.ResourceData, meta interface{}) error {
 	if d.IsNewResource() {
 		if props := read.Properties; props != nil {
 			if vault := props.VaultURI; vault != nil {
-				log.Printf("[DEBUG] Waiting for Key Vault %q (Resource Group %q) to become available", name, resGroup)
-				stateConf := &resource.StateChangeConf{
```

Comment:
I really don't recall changing this at all.  I think this was pulled in from master at some point.  Because I don't know why I would have changed this at all.  I will change this to be back in line with master